app/models/volunteer.rb

```diff
@@ -2,6 +2,19 @@
 class Volunteer < ActiveRecord::Base
   default_scope { order('volunteers.name ASC').where(active: true) }
 
+  scope :in_regions, ->(regions) { joins(:regions).where(regions: { id: Array.wrap(regions) }).uniq }
```

Comment:
`scope :in_regions, ->(region_ids) { joins(:regions).where(regions: { id: region_ids }) }` How do we feel about that instead?

Reply:
Good news! Mine is more correct. The old `needs_training?` method applies the `where(active: true)` scope when using the logs relation and this accounts for all 6 volunteers. They all had "active=false" LogVolunteer records for region 1, so they showed up as needing training even though they had completed Logs.

I would assume they don't need training even if they are inactive. As long as they have completed something

https://github.com/boulder-food-rescue/food-rescue-robot/blob/master/app/models/volunteer.rb#L22-L23